### PR TITLE
feat: jobs page

### PR DESCRIPTION
**Pull request by maban**
This is a very basic jobs landing page. **The content is dummy** – it would be nice to have a short teaser for each role, at least saying the location, but I've made the engineering one more generic since that changes more often.

It's very basic, but I think it will do for now.

screen shot 2017-03-27 at 12 07 06

### CODE REVIEW

File: jobs.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+image: "/"
+
+---
+
+
+Snyk is on a mission to secure the way people develop and consume open source. We analyse and protect your projects, eliminating known vulnerabilities and securing applications in runtime.
+
+---
+
+## [Head of Product]()
```
Comment (joshje) [inline jobs.html]: Is this a link to nowhere? Should it link to the job description?

File: jobs.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+# These are used for social media cards, it is important they are filled out.
+meta-description: "Meta description"
+excerpt: "Description of what this page is, for social media cards"
+image: "/"
+
+---
+
+
+Snyk is on a mission to secure the way people develop and consume open source. We analyse and protect your projects, eliminating known vulnerabilities and securing applications in runtime.
```
Comment (geva) [inline jobs.html]: Snyk's mission is to help developers and organizations use open source and say secure. Snyk is used by small and large organizations alike to find, fix (and prevent!) vulnerabilities in open source dependencies as part of a CI/CD pipeline. Interested in joining the team? See below for open jobs!

File: jobs.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+
+## [Head of Product]()
+
+London or Tel Aviv
+
+---
+
+## [Sales]()
```
Comment (geva) [inline jobs.html]: Make the 'Sales' a header like the Engineering, and then inside make an "Account Executive - San Francisco Bay Area or London" line item. The Account Executive should link to https://www.linkedin.com/job/cap/overview/index/300497462

File: jobs.html (hunk 1 above)
Comment (geva) [inline jobs.html]: See note on Sales below. Make Product same - "Product" as category header, and line item with head of product linking to GH job desc

**Pull request merged**